### PR TITLE
Oro version updated to 5.0

### PR DESCRIPTION
**Pull request by Zigr1**

### CODE REVIEW

**Comment by Zigr1** [comment]
Might need additional fixes after testing in Oro 5.0

File: composer.json [hunk 1]
```diff
@@ -31,9 +31,9 @@
   "minimum-stability": "dev",
   "prefer-stable": true,
   "require": {
-    "oro/platform": "~4.2.0",
+    "oro/platform": "5.0.*",
     "gaufrette/aws-s3-adapter": "^0.4.0",
-    "knplabs/gaufrette": "0.9.0",
+    "knplabs/gaufrette": "0.10.0",
```
Comment (AdamJHall) [inline composer.json]: @Zigr1, Should we up this higher? look `0.11.1` was released and PHP 8.0 support was technically only added in `0.11.0`

**Review by AdamJHall — CHANGES_REQUESTED** [review]

File: composer.json (hunk 1 above)
Comment (Zigr1) [reply composer.json]: Yeah, that's certainly meant to be 11. I believe it was a typo

**Review by AdamJHall — APPROVED** [review]
As discussed in Slack, happy to keep at this level if that is what Oro is sticking to.

**Pull request merged**